.vscode/settings.json

```diff
@@ -1,6 +1,15 @@
 {
+  "search.exclude": {
+    "Library/Homebrew/vendor/bundle/ruby/": true,
+    "Library/Homebrew/vendor/gems/": true,
+    "Library/Homebrew/vendor/portable-ruby/": true
+  },
   "editor.insertSpaces": true,
   "editor.tabSize": 2,
+  "editor.rulers": [
+    80,
```

Comment:
What is the `80` for? If it's not meant for Ruby files, can we nest these under the language specific blocks so we don't have extra lines

Reply:
@Rylan12 we have a non-trivial amount of times that we're trying to output e.g. a message that's wrapped to `80`. This don't do wrapping but will show this line here.

Will merge as-is given the ✅ but definitely open to someone removing it in a PR if it's too annoying.